server/models/welcomeMessages.js

```diff
@@ -32,6 +32,7 @@ const WelcomeMessages = {
 
       // Create new messages
       for (const [index, message] of messages.entries()) {
+        if (!message.response) continue;
```

Comment:
There are a few options here to optimize the code. Doing an insert on each message is really not ideal and I didn't see we were doing that.

```
    const data = messages
      .filter(message => message.response)
      .map((message, index) => ({
        user: message.user,
        response: message.response,
        orderIndex: index,
      }));

    await prisma.welcome_messages.createMany({ data });
 ```
 
 Both filters and upserts in a single call. I would test to see if that above chunk works and if does, id be much happier with that solution.